paddle/fluid/framework/grad_op_desc_maker.h

```diff
@@ -132,22 +141,47 @@ class GradOpDescMakerBase {
   std::string ForwardOpType() const { return this->fwd_op_.Type(); }
 
  protected:
-  const OpDesc& ForwardOp() const { return fwd_op_; }
+  // const OpDesc& ForwardOp() const { return fwd_op_; }
```

Comment:
Remove comments if the method is not necessary.

Reply:
done